packages/migration/src/migrations/v0042_removeTrackName.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import type {
+  GifResourceV41,
+  ImageResourceV41,
+  VideoResourceV41,
+  GifElementV41,
+  ImageElementV41,
+  PageV41,
+  ProductElementV41,
+  ShapeElementV41,
+  StoryV41,
+  TextElementV41,
+  VideoElementV41,
+} from './v0041_removeFontProperties';
+
+// @todo Should we now add Track version to each file instead?
```

Comment:
Waiting for feedback here, too -- `Track` is changed twice, should we still version it in each file?

Reply:
Seems like a lot of repetition. If we can avoid that, why not :)